src/lib.rs

```diff
@@ -242,18 +242,20 @@ A: Sized + ApplicationBase<gfx_device_dx11::Resources, D3D11CommandBuffer>
             }
         });
         if let Some((width, height)) = new_size {
-            use gfx_window_dxgi::update_views;
-            match update_views(&mut window, &mut factory, &mut device, width, height) {
-                Ok(new_color) => {
-                    let new_depth = factory.create_depth_stencil_view_only(width, height).unwrap();
-                    app.on_resize(&mut factory, WindowTargets {
-                        color: new_color,
-                        depth: new_depth,
-                        aspect_ratio: width as f32 / height as f32,
-                    });
-                },
-                Err(e) => error!("Resize failed: {}", e),
-            }
+            //TODO: clean up the old references first!
```

Comment:
How do you plan to do that ? Some interior mutability might enable the DX11 backend to update the existing reference or to invalidate it. Another possibility could be to have some kind of generation id to avoid reference mutation while still invalidating it.

Reply:
Sure, good ideas!
It's not a big priority for me now to make this work universally on Dx11. The client doesn't use `gfx_app`, they just need a way to read back textures.